src/style.css

```diff
@@ -0,0 +1,105 @@
+/* stylelint-disable color-hex-length */
+/* stylelint-disable color-hex-case */
+/* stylelint-disable string-quotes */
+
+@font-face {
+  font-family: 'Avenir';
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir';
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+}
+
+html {
+  font-family: 'Avenir', sans-serif;
+}
+
+.one {
+  width: 100%;
+  height: 70px;
+  box-sizing: border-box;
+  display: block;
+  border: 1px solid #E1E7ED;
+  border-radius: 4px;
+  color: #3D4E61;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  margin-top: 20px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+  font-family: inherit;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: 26px center;
+}
+
+.one:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  font-family: inherit;
+  font-weight: 300;
+}
+
+.one:focus {
+  background: linear-gradient(180deg, #FFFFFF 0%, #F6F6F7 100%);
+  border-radius: 3px;
+  outline: 0;
+  font-family: inherit;
+  font-weight: 700;
+}
+
+.two {
```

Comment:
You have a lot of repeatable styles. 
So you need to refactor you code. I will give you an EXAMPLE, how to do it

Following the BEM-methodology - use the `block` selector for the FIRST form (`class="form"`), where you apply common form styles

```
.form {
  margin: 20px 0;
  border: solid 1px #e1e7ed;
  border-radius: 4px;
  height: 70px;
  position: relative;
  margin-top: 20px;
}
```

And then you apply `modifier` to the SECOND form (`class="form form--small"`) that have ONLY differences with base class form
```
.form--small {
  height: 42px;
  border-radius: 4px;
}
```
So in this way you follow DRY (Don't repeat yourself) principle
Apply this principe all over the styles